index.md

```diff
@@ -200,6 +200,13 @@ verification approach:
   OOPSLA, 2012.
   - This paper presents a verification framework that is parametric in a (trusted) operational semantics of some programming language. The underlying proof system is language-independent and consists of eight proof rules. The proof system is proved partially correct and relatively complete (with respect to the programming language configuration model). To show its practicality, the generic framework is instantiated with a fragment of C and evaluated with encouraging results.
 
+## Community
+
+There is no official place to discuss matching logic. Here we list the options that we are trying to follow.
```

Comment:
How about making the tone more positive? For example, "We welcome any comments, suggestions, and questions! Feel free to reach out to us at the following options:" or something alike.

Reply:
You're right! I was in a hurry with the phrasing... How about this? I tried to use your idea with the fact that we're trying to build a community. Not sure if it works.

Reply:
@xc93  Can you please approve? Thank you.